content/documentation/edge/installation-guide/index.adoc

```diff
@@ -83,7 +83,7 @@ If you are installing on OpenShift, you must grant the `cluster-admin` role to t
 
 == Install Kiali via Istio or Maistra
```

Comment:
While you're in the install guide you may as well update the Version Compatibility section above.   Istio 1.7+ can set Kiali 1.22+ with a note about the installation change.

Reply:
I'm adding this:
```
| 1.7.0+
| 1.22.0+
| Istio 1.7 istioctl will no longer install Kiali. Use the Istio samples/addons all-in-one yaml or the Kiali Helm Chart.
```